extensions/ql-vscode/src/remote-queries/variant-analysis-manager.ts

```diff
@@ -234,7 +257,7 @@ export class VariantAnalysisManager extends DisposableObject implements VariantA
    * used by the query history manager to determine when the directory
    * should be deleted.
    */
-  private async prepareStorageDirectory(variantAnalysisId: number): Promise<void> {
+  public async prepareStorageDirectory(variantAnalysisId: number): Promise<void> {
```

Comment:
I don't really think this should be public since it's quite specific to the manager. I'm not sure how we can improve this though, but since this is public to be used in tests I think we can just copy the code to the tests.

Reply:
If you think it's better to keep this method private but duplicate the implementation in the tests, I'm happy to go with that approach.